examples/tuning/cta_svm/config_yamls/2nd_test_acc_params_tuning_config.yaml

```diff
@@ -0,0 +1,65 @@
+type: preprocessor
```

Comment:
Under the second highest configuration of test_acc in step2, the parameter range in step3